SwrveSDK/src/google/java/com/swrve/sdk/config/SwrveConfig.java

```diff
@@ -62,4 +67,18 @@ public boolean isGAIDLoggingEnabled() {
     public void setGAIDLoggingEnabled(boolean enabled) {
         this.gAIDLoggingEnabled = enabled;
     }
+
+    /**
+     * @return if it will automatically obtain the Google Cloud Messaging Registration ID.
+     */
+    public boolean isObtainRegistrationIdEnabled() {
```

Comment:
No, not both. However, it is an async process so we can't assume it will be ready on SDK start so that we will know we don't have to ask for it ourselves.

It is indeed not the best name, what are your suggestions? 'automaticPushRegistration'?